SAPHana/ra/SAPHanaController

```diff
@@ -2692,10 +2798,16 @@ function saphana_promote_clone() {
                             lpa_push_lpt $LPTloc
                             backup_global_and_nameserver
                             set_hana_attribute "$NODENAME" "$sr_name" "${ATTR_NAME_HANA_PRIM[@]}"
-                            # TODO PRIO5: 122 should not be static but come from the scoring table
-                            set_crm_master 122
+                            # TODO PRIO5: 145 should not be static but come from the scoring table
+                            set_crm_master 145
                             set_hana_attribute "$NODENAME" "-" "${ATTR_NAME_HANA_SEC[@]}"
-                            su - $sidadm -c "hdbnsutil -sr_takeover"
+                            set_SRHOOK "$sr_name" "PRIM"
+                            super_ocf_log info "ACT: SAPHANA TAKEOVER: SET MARK"
+                            set_hana_attribute "$NODENAME" "T" "${ATTR_NAME_HANA_SRACTION[@]}"
+                            su - $sidadm -c "hdbnsutil -sr_takeover"; srt_rc=$?
+                            super_ocf_log info "ACT: SAPHANA TAKEOVER: srt_rc=$srt_rc"
+                            set_hana_attribute "$NODENAME" "-" "${ATTR_NAME_HANA_SRACTION[@]}"
```

Comment:
Gleiche Frage wie schon zuvor:
Wenn sr_takeover einen Fehler liefert (rc !=0), sollte dann das Attribut auf '-' gesetzt werden oder könnte man dann hier ein anderes Attribut setzten, um diesen Fehler auch in den Attributen zu reflektieren?
Für die Anzeige bei 'show-attribute'

Reply:
Wie oben.